app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsAdapter.kt

```diff
@@ -1,26 +1,40 @@
 package ar.com.wolox.android.example.ui.home.news
 
+import android.graphics.Typeface
 import android.view.LayoutInflater
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
 import ar.com.wolox.android.R
 import ar.com.wolox.android.example.model.News
 
-class NewsAdapter(private val news: List<News>) : RecyclerView.Adapter<NewsViewHolder>() {
+class NewsAdapter(private val news: ArrayList<News>, private val userId: Int) : RecyclerView.Adapter<NewsViewHolder>() {
+    private var newsList = news
+    private var currentUserId = userId
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): NewsViewHolder {
         val view = LayoutInflater.from(parent.context)
                 .inflate(R.layout.view_new_item, parent, false)
         return NewsViewHolder(view)
     }
 
-    override fun getItemCount(): Int = news.size
+    override fun getItemCount(): Int = newsList.size
 
     override fun onBindViewHolder(holder: NewsViewHolder, position: Int) {
-        with(news.elementAt(position)) {
-            holder.newsTitle.text = title
-            holder.newsDescription.text = description
-            if (isLiked) holder.likeButton.setImageResource(R.drawable.ic_like_on)
-            else holder.likeButton.setImageResource(R.drawable.ic_like_off)
+        with(newsList.elementAt(position)) {
+            holder.newsTitle.setTypeface(null, Typeface.BOLD)
+            holder.newsTitle.text = commenter
+            if (comment.length > MAX_COMMENT_LENGTH) holder.newsDescription.text = """${comment.substring(0, MAX_COMMENT_LENGTH)} ..."""
```

Comment:
You can achieve this by adding `maxLenght="x"` and `ellipsize="end"` in the xml file